packages/frontend/core/src/components/affine/auth/oauth.tsx

```diff
@@ -110,7 +110,10 @@ interface OauthProviderProps {
 }
 
 function OAuthProvider({ onContinue, provider }: OauthProviderProps) {
-  const { icon } = OAuthProviderMap[provider];
```

Comment:
The logic here should be refactored; if the provider map does not exist, the login button should not be displayed

Reply:
It seems that even it not in oauth provider map, oauth can still work, just no icon